### ISSUE TITLE
Chinese not showing up in Markdown Preview

### ISSUE THREAD

**Issue by mingjunyang** [atom/markdown-preview#238]
My OS openSUSE 13.2 (Harlequin) (x86_64) 64 位 with GNOME 3.14.2
Atom is 0.190.0
markdown-preview 0.148.0

```markdown
#ymsWebClient



###build dist
 1. 手工删当前的额dist文件夹

 2. 项目根目录下执行命令
  `
  gulp build
  `
```
```html
<h1 id="ymswebclient">ymsWebClient</h1>
<h3 id="build-dist">build dist</h3>
<ol>
<li><p>手工删当前的额dist文件夹</p>
</li>
<li><p>项目根目录下执行命令
<code style="font-family: DejaVu Sans Mono,Source Code Pro,WenQuanYi Micro Hei;">gulp build</code></p>
</li>
</ol>
```

But privew is ![image](https://raw.githubusercontent.com/mingjunyang/myvimconf4c/master/2015-04-08.png)
config.cson
```cson
"*":
  "exception-reporting":
    userId: "3c3323ae-d7b7-8749-bdce-32c65bf202da"
  welcome:
    showOnStartup: false
  core:
    disabledPackages: [
      "welcome"
      "language-r"
      "javascript-snippets"
      "git-diff-details"
    ]
    themes: [
      "atom-dark-ui"
      "solarized-dark-syntax"
    ]
    projectHome: "/home/mingjunyang/git"
  editor:
    invisibles: {}
    showIndentGuide: true
    tabLength: 4
    zoomFontWhenCtrlScrolling: false
    softWrap: true
    fontFamily: "DejaVu Sans Mono,Source Code Pro,WenQuanYi Micro Hei"
    fontSize: 22
  "atom-beautify":
    _analyticsUserId: "9821831b-a2aa-4903-a52f-3599aea0f6c0"
    js_keep_function_indentation: true
    css_newline_between_rules: true
    css_selector_separator_newline: true
  minimap:
    plugins:
      "git-diff": false
  "language-context-free":
    variation: "A"
  "todo-show":
    ignoreThesePaths: [
      "/node_modules/"
      "/.git/"
      "/app/bower_components/"
      "/app/dist/"
    ]
  "tree-view":
    hideVcsIgnoredFiles: true
  "css-color-highlight": {}
  "atom-color-highlight": {}
  "autocomplete-plus": {}
  "autocomplete-clang":
    preCompiledHeaders: {}
    std: {}
  "markdown-preview": {}
```

**Comment by mnquintana** [comment]
@mingjunyang Can you try uploading the screenshot again? (It's currently broken.) You can upload it by dragging and dropping it into the issue's edit text box.

**Comment by wong2** [comment]
what do you get if you copy the html code into a .html file and open it in browser?

**Comment by mingjunyang** [comment]
@wong2  save as a.html  and copy as html , open in  browser is good work.正常显示。

**Comment by 50Wliu** [comment]
Refs atom/atom#6214.

**Comment by mengzyou** [comment]
Subscribe this issue, I have the same issue on **v0.191.0** .

**Comment by superlin** [comment]
Will it support Chinese in the future?

![onpj31b x ea9g0o_ qj](https://cloud.githubusercontent.com/assets/3763687/7441785/8093e370-f129-11e4-8e5b-fc313ea80173.png)

**Comment by 50Wliu** [comment]
@superlin have you tried 0.196.0?

**Comment by lcgong** [comment]
I have  also  the same issue on 0.196.

I tried atom/atom#6214. But it does not works. Where should I set font family for markdown-preview.

**Comment by mycstdlib** [comment]
I found that turning the Themes into 'Atom Light' will fix this problem on my machine.
Settings->Themes->change UI Theme&Syntax Theme both to 'Atom Light'.
My atom version is 0.199
Maybe you can try it.

**Comment by mengzyou** [comment]
@mycstdlib this workaround is not work for me on openSUSE 13.2.

**Comment by frantic1048** [comment]
@mengzyou ensure you have  `文泉驿等宽微米黑` font on your machine, or any other fonts support Chinese  are ok (such as `Adobe Source Han Sans`).

simply you can try:

    .markdown-preview{
      font-family: "文泉驿等宽微米黑" !important;
    }

**Comment by mengzyou** [comment]
@frantic1048 That's great, works fine, thank you very much!

**Comment by mingjunyang** [comment]
I do it ,It's may be okay.
```css
.markdown-preview{
  font-family: "WenQuanYi Micro Hei" !important;
}
```

**Comment by greenqy** [comment]
@mingjunyang this works fine for me, thank you!

**Comment by abcfy2** [comment]
Thank to @mingjunyang , it works.

**Comment by kurtzhong** [comment]
@mingjunyang  Thanks. Adding the custom style does solve my issue on `v1.0.11` on Fedora 22:  
```css
.markdown-preview
{
  font-family: "文泉驿等宽微米黑" !important;
}
```
But first ensure the fonts mentioned is installed using: 

```bash
sudo dnf install wqy* -y
```

**Comment by mingjunyang** [comment]
News!!!
Today,I update atom and I remove the file `~/.atom/style.less`,maybe it's okay.
```text
1.0.13 2015/9/18 上午8:49:51

Notable Changes

Improved the performance of detecting incompatible native modules upon upgrading to a new Electron version.
1.0.12 2015/9/18 上午3:07:17

Notable Changes

Fixed an issue where file names with line numbers (e.g. file.text:50) were sometimes handled incorrectly.
Text editors' undo histories are now preserved when windows are reloaded.
Atom now ships with Chrome 43 and io.js 2.3.1.
```

**Comment by 50Wliu** [comment]
Can anyone else try updating to Atom 1.0.12 or above and seeing if that fixes their issue?  Thanks.

**Comment by 50Wliu** [comment]
Going to close this since it should have been fixed in Atom 1.0.12 and above.  If you're still experiencing this even on Atom 1.0.12, please comment below.

**Comment by abcfy2** [comment]
Yes, it's fixed since 1.0.13. I've upgraded to the latest new version, and this issue fixed.
